File: Graphs/Node.java
package Graphs;

public class Node
{
    private String Name;
    public boolean connected;
    private Node connectedto;



public Node(){
    

}

    public void setconnectedto(Node connectedto){
        this.connectedto = connectedto;
    }
    
    public Node getconnectedto(){
        return connectedto;
    }
    
 
public void setconnected(boolean  value){
    connected = value;
}

public boolean getconnected(){
    return connected;
}

public String getName(){
    return Name; 
}

public void SetName(String _Name){
this.Name = _Name;
}





}

File: Graphs/Node.py
class Node:
    def __init__(self,name,age):
        self.name = name
        self.age = age 
        self.neighbours = []
        self.connected = False
        self.connectedto = self

    def add_neighbour(self,neighbour):
        self.neighbours.append(neighbour)
        print(self.neighbours)

    def setconnectedto(self,node):
        self.connectedto = node
        print(self.connectedto)


    
